app/src/main/res/values/dimens.xml

```diff
@@ -214,157 +50,350 @@
   <dimen name="ratio_input_interaction_item_non_conversation_view_margin_end">32dp</dimen>
+  <dimen name="text_input_interaction_item_non_conversation_view_margin_start">32dp</dimen>
+  <dimen name="text_input_interaction_item_non_conversation_view_margin_end">32dp</dimen>
+  <dimen name="text_input_interaction_item_margin_start">80dp</dimen>
+  <dimen name="numeric_input_interaction_item_conversation_view_margin_start">40dp</dimen>
+  <dimen name="ratio_input_interaction_item_conversation_view_margin_start">40dp</dimen>
+  <dimen name="text_input_interaction_item_conversation_view_margin_start">40dp</dimen>
+  <dimen name="ratio_input_interaction_item_conversation_view_margin_end">24dp</dimen>
+  <dimen name="text_input_interaction_item_conversation_view_margin_end">24dp</dimen>
+  <dimen name="numeric_input_interaction_item_conversation_view_margin_end">24dp</dimen>
+
+  <!--Fraction Interactions-->
+  <dimen name="fraction_interaction_item_non_conversation_view_margin_start">32dp</dimen>
+  <dimen name="fraction_interaction_item_non_conversation_view_margin_end">32dp</dimen>
+  <dimen name="fraction_interaction_item_conversation_view_margin_start">40dp</dimen>
+  <dimen name="fraction_interaction_item_conversation_view_margin_end">24dp</dimen>
+
+  <!--Selection Interactions-->
   <dimen name="selection_interaction_item_non_conversation_view_margin_start">32dp</dimen>
   <dimen name="selection_interaction_item_non_conversation_view_margin_end">32dp</dimen>
+  <dimen name="selection_interaction_item_conversation_view_margin_start">40dp</dimen>
+  <dimen name="selection_interaction_item_conversation_view_margin_end">24dp</dimen>
+
+  <!-- Answer-->
+  <dimen name="answer_tick_size">24dp</dimen>
+  <dimen name="answer_tick_margin">12dp</dimen>
+  <dimen name="submitted_answer_text_non_conversation_view_padding_top">8dp</dimen>
+  <dimen name="submitted_answer_text_non_conversation_view_padding_bottom">8dp</dimen>
+  <dimen name="submitted_answer_text_conversation_view_padding_bottom">16dp</dimen>
+  <dimen name="submitted_answer_text_conversation_view_padding_top">16dp</dimen>
+
+  <!--Audio-->
+  <dimen name="audio_fragment_corner_radius">8dp</dimen>
+  <dimen name="audio_fragment_margin">28dp</dimen>
+
+  <!--Recently Played-->
+  <dimen name="recently_played_margin_max">28dp</dimen>
+  <dimen name="recently_played_margin_min">8dp</dimen>
+  <dimen name="recently_played_tablet_land_outer_margin">96dp</dimen>
+  <dimen name="recently_played_tablet_port_outer_margin">120dp</dimen>
+
+  <!--Font Scale-->
+  <dimen name="font_scale_content_extra_large_text_view_size">28dp</dimen>
+  <dimen name="font_scale_content_text_size">20dp</dimen>
+  <dimen name="font_scale_content_small_text_view_size">16dp</dimen>
+  <dimen name="font_scale_content_large_text_view_size">24dp</dimen>
+
+  <!-- Fade -->
+  <integer name="fade_in_duration_ms">1000</integer>
+  <integer name="fade_out_duration_ms">1000</integer>
+
+  <!--State-->
```

Comment:
Nit update everywhere:
There should be space before and after words.
`<!-- State -->`